index.html

```diff
@@ -1,55 +1,247 @@
+        </ul>
+        <div class="handburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
     </header>
 
     <section class="first-section">
-        <div class="container">
-            <h1>Hey there. I’m Daniel</h1>
-            <h1 class="desc">I’m a software developer</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-                <ul class="social">
-                    <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
-                    <li><a href="#"> <img src="./Images//linkedin.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images/medium.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
-                </ul>
-        </div>
+      <div class="container">
+        <h1>Hey there. I’m Daniel <br> <span class="desc">I’m a software developer</span> </h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social">
+          <li>
+            <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//github.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+          </li>
+        </ul> 
+      </div>
     </section>
-    <!-- <button><img src="./Images//disabled.svg" alt=""></button>
+    <button ><img class="down-icon" src="./Images//disabled.svg" alt="" /></button>
     <section class="second-section">
+      <div class="container">
+        <h2 class="recent-header">My Recent Works</h2>
+        <hr />
+        <div class="boxes">
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h2 class="title">Multi-Post Stories <br>  Gain+Glory</h2>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <div><button class="box-button">See Project</button></div>
+            </div>
```

Comment:
Please consider making the cards with `.box class ` on them responsive as their width still is not responsive ( Does not increase or decrease in accordance with the change of viewport width ) You can achieve it by applying the `min-width: 327px`